packages/node_modules/nav-frontend-skjema/src/valid-input.js

```diff
@@ -0,0 +1,286 @@
+                }));
+            } else if (this.props.min) {
+                this.validators.push(() => ({
+                    type: 'isGreaterThan',
+                    verdict: this.validator.isGT(parseInt(this.props.min, 10))
+                }));
+            } else if (this.props.max) {
+                this.validators.push(() => ({
+                    type: 'isLessThan',
+                    verdict: this.validator.isLT(parseInt(this.props.max, 10))
+                }));
+            }
+        }
+
+        if (this.props.maxLength && this.props.minLength) {
+            this.validators.push(() => ({
+                type: 'hasLengthRange',
+                verdict: this.validator.hasLengthRange(this.props.minLength, this.props.maxLength)
+            }));
+        } else if (this.props.maxLength) {
+            this.validators.push(() => ({
+                type: 'isShorterThan',
+                verdict: this.validator.isShorterThan(this.props.maxLength)
+            }));
+        } else if (this.props.minLength) {
+            this.validators.push(() => ({
+                type: 'isLongerThan',
+                verdict: this.validator.isLongerThan(this.props.maxLength)
+            }));
+        }
+
+        if (type === 'email') {
+            this.validators.push(() => ({
+                type: 'isEmailFormat',
+                verdict: this.validator.isEmailFormat('gi')
+            }));
+        }
+
+        if (type === 'fnr') {
+            this.validators.push(() => ({
+                type: 'isBirthNumFormat',
+                verdict: this.validator.isBirthNumFormat()
+            }));
+        }
+
+        if (type === 'accountnr') {
+            this.validators.push(() => ({
+                type: 'isAccountNumFormat',
+                verdict: this.validator.isAccountNumFormat()
+            }));
+        }
+
+        if (type === 'orgnr') {
+            this.validators.push(() => ({
+                type: 'isOrgNumFormat',
+                verdict: this.validator.isOrgNumFormat()
+            }));
+        }
+
+        if (this.props.matchPattern && type !== 'number') {
+            this.validators.push(() => ({
+                type: 'pattern',
+                verdict: this.validator.matchesPattern(this.props.matchPattern, 'gi')
+            }));
+        }
+
+        // If there is a custom validation function defined, add it to the validators array
+        if (this.props.custom && typeof this.props.custom === 'function') {
+            this.validators.push(() => ({
+                type: 'custom',
+                verdict: this.props.custom(this.element.value)
+            }));
+        }
+    }
+
+    onChange(e) {
+        if (this.props.validateOnChange) {
+            this.validate();
+        }
+        if (this.hasCustomOnChange) {
+            this.props.onChange(e);
+        }
+    }
+
+    onBlur(e) {
+        if (this.props.validateOnBlur) {
```

Comment:
validering må alltid kjøres, sånn sett er det ikke behov for å legge muligheten for å skru det av.